### ISSUE TITLE
How to convert MKV to WEBM format

### ISSUE THREAD

**Issue by unite-deals** [dgasaway/towebm#1]
Could you able to provide an example .
Btw is there any syntax , maximum type video convert into webm .
Thanks in advance

**Comment by dgasaway** [comment]
There are many examples given in the README.  Is there something specific you want to know?

**Comment by unite-deals** [comment]
Yeah I have seen that , but I have not found any code blocks about mkv to webm , could you you able to guide to me ?

**Comment by dgasaway** [comment]
There are several examples in the README with an mkv input file.  However, it really should work with any kind of input video which ffmpeg can read, which is many different formats.  I'd like to help, but I'm not sure what help you need.  Tell me exactly what you are trying to do.
